conda/shell/condabin/conda.bat

```diff
@@ -5,7 +5,11 @@
 @IF [%1]==[activate]   "%~dp0_conda_activate" %*
 @IF [%1]==[deactivate] "%~dp0_conda_activate" %*
 
+setlocal
+for %%B in (%~dp0.) do set PATH=%%~dpB;%%~dpBLibrary\mingw-w64\bin;%%~dpBLibrary\usr\bin;%%~dpBLibrary\bin;%%~dpBScripts;%%~dpBbin;%PATH%
```

Comment:
Probably need quoting?

```
for %%B in ("%~dp0.") do set "PATH=%%~dpB;%%~dpBLibrary\mingw-w64\bin;%%~dpBLibrary\usr\bin;%%~dpBLibrary\bin;%%~dpBScripts;%%~dpBbin;%PATH%"
```

Reply:
Nope.

Reply:
Neither need quoting here. BBIAB, going for a little walk.

Reply:
It needs quoting, otherwise causing problems if the path contains spaces. The loop iterates over each substring separated by space.
Also, in conda_hook.bat there is something very similar:
@FOR %%F in ("%~dp0") do @SET "__condabin_dir=%%~dpF"

Reply:
Checking, thanks @czatom.

Reply:
Latest version is at  https://github.com/mingwandroid/conda/blob/ray_activation_fixes/conda/shell/condabin/conda.bat#L12-L17

While we are now quoting it prompting me to check the code found a pointless, broken for loop, so thanks again.